### PR TITLE
Adds friendly_id for slugs.

### PR DESCRIPTION
**Pull request by BriocheBerlin**
- Needs tests. Homework for Sara and Brigitte ;)
- On depolyment run:  Group.find_each(&:save) and  Post.find_each(&:save)

### CODE REVIEW

**Comment by sareg0** [comment]
@BriocheBerlin @til und ich, made the tests, and the WORK!! Is this a thing we can merge now?

**Comment by til** [comment]
Meh, not mergeable yet, because due to the slugs changes, three other specs are now failing. It could be because group_path is now returning the slugified URL, but I am not sure.

Those specs need to be fixed first. Maybe they are pointing to a real problem that the slugs functionality introduced, or maybe it's just the specs that got irritated.

**Comment by sareg0** [comment]
Dang, ok. Well that will be our job for next week then. Should have run all the tests I guess :)

**Comment by berlintam** [comment]
@til @BriocheBerlin look! it works!

File: lib/tasks/set_slugs.rake [hunk 1]
```diff
@@ -0,0 +1,9 @@
+task set_slugs: :environment do
```
Comment (berlintam) [inline lib/tasks/set_slugs.rake]: this needs to be run on the server when deployed!

**Comment by berlintam** [comment]
oops, nevermind. still some failing specs :(

**Comment by berlintam** [comment]
OMG!

breakingbad

**Comment by sareg0** [comment]
#yolomerge

**Pull request merged**